examples/Makefile.am

```diff
@@ -59,9 +59,11 @@ uct_hello_world_LDADD    = $(top_builddir)/src/ucs/libucs.la \
                            $(EXAMPLE_CUDA_LIBS)
 
 ucp_client_server_SOURCES  = ucp_client_server.c
-ucp_client_server_CFLAGS   = $(BASE_CFLAGS) $(CFLAGS_PEDANTIC)
-ucp_client_server_CPPFLAGS = $(BASE_CPPFLAGS)
+ucp_client_server_CFLAGS   = $(BASE_CFLAGS) $(EXAMPLE_CUDA_CFLAGS)
+ucp_client_server_CPPFLAGS = $(BASE_CPPFLAGS) $(EXAMPLE_CUDA_CPPFLAGS)
+ucp_client_server_LDFLAGS  = $(EXAMPLE_CUDA_LD_FLAGS)
```

Comment:
how does it work if we install client server example as binary package? i assume it will not run if compiled without cuda?
need to build separate binaries for examples with cuda support and install them as part of ucx-cuda subpackage